locales/en/moves.json

```diff
@@ -114,5 +114,8 @@
       "content": "Reference number<br><strong>{{moveReference}}</strong>"
     },
     "detail": "The move for <strong><a href=\"{{href}}\">{{name}}</a></strong> to <strong>{{location}}</strong> on <strong>{{date}}</strong> has been scheduled."
+  },
+  "reason": {
+    "heading": "Request type"
```

Comment:
Could keep the consistency of using the question as the heading like we have on the other steps?

Reply:
In the prototype, the heading has different wording from the question.

Reply:
Might be worth still considering this. But we can review when we look at the journey as a whole afterwards maybe.